servers/zms/src/main/java/com/yahoo/athenz/zms/DBService.java

```diff
@@ -1838,7 +1838,7 @@ DomainRoleMembers listDomainRoleMembers(String domainName) {
         }
     }
 
-    Role getRole(String domainName, String roleName, Boolean auditLog, Boolean expand, Boolean pending) {
+    public Role getRole(String domainName, String roleName, Boolean auditLog, Boolean expand, Boolean pending) {
```

Comment:
any reason we made this and the getAthenzDomain method public?

Reply:
Because this is getting called from within NotificationManager now.  It had package access before.